Dockerfile

```diff
@@ -12,19 +12,23 @@
 # See the License for the specific language governing permissions and
 # limitations under the License.
 
-FROM debian:stretch
+FROM golang:1.11-stretch
```

Comment:
Is this necessary? I would prefer full control over the go version. No go version anywhere in the path.

Reply:
ack. will play with this image and then try going back to debian:stretch

Reply:
@sttts in init-repo/main.go, we try to install `github.com/tools/godep` and `github.com/golang/dep` for which we need a golang present in the container. right?